src/main/resources/templates/users/view-signature.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE HTML>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
+          integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+
+    <title>View Signature </title>
+</head>
+
+<body>
+<p th:text="'User: ' +  ${user.firstName} + ' '+ ${user.lastName}"></p>
+<img th:src="@{'data:image/jpg;base64,'+${user.getEncodedSignature()}}"/>
```

Comment:
See comment above - this works just fine if we want to mix Thymeleaf templates and our SPA frontend.